spec/01-unit/08-router_spec.lua

```diff
@@ -1732,6 +1732,69 @@ for _, flavor in ipairs({ "traditional", "traditional_compatible", "expressions"
         end)
       end)
 
+      if flavor == "tradition_compatible" then
```

Comment:
I think that now this feature also should support 'expressions' flavor,
could you check the test cases to ensure it?
```suggestion
      if flavor == "tradition_compatible" and flavor == "expressions" then
```

Reply:
Sure. Added "expressions" flavor.